src/expression/definitions/format.ts

```diff
@@ -93,15 +105,23 @@ export class FormatExpression implements Expression {
         const evaluateSection = section => {
             const evaluatedContent = section.content.evaluate(ctx);
             if (typeOf(evaluatedContent) === ResolvedImageType) {
-                return new FormattedSection('', evaluatedContent, null, null, null);
+                return new FormattedSection(
+                    '',
+                    evaluatedContent,
+                    null,
+                    null,
+                    null,
+                    section.verticalAlign ? section.verticalAlign.evaluate(ctx) : null
```

Comment:
How is this different than other properties? Why not passing null here?

Reply:
This is the instantiation of `FormattedSection` for image. All other properties are null, because they only applies to text (`font` etc.), but `vertical-align` can be applied to both image and text section.